app/src/test/java/org/oppia/android/app/testing/options/AppLanguageFragmentTest.kt

```diff
@@ -104,6 +112,19 @@ class AppLanguageFragmentTest {
     Intents.release()
   }
 
+  @Test
+  fun testAppLanguageActivity_hasCorrectActivityLabel() {
+    activityTestRule.launchActivity(
```

Comment:
Could you please add a screenshot showing this test is passing on espresso?

Reply:
I haven't done much in testing. Can you please guide me on how can I run these tests on espresso or do I need to write new tests for espresso?

Reply:
@abhinav12k https://github.com/oppia/oppia-android/wiki/Oppia-Android-Testing

Reply:
Do we require this test here or are we removing this as we have a new file now?

Reply:
No we can remove the tests from here